Wikipedia/Code/WKWebView+ElementLocation.m

```diff
@@ -54,8 +54,8 @@ - (void)getScreenRectForHtmlElementFromJavascriptString:(NSString *)javascriptSt
            }];
 }
 
```

Comment:
This is so we're doing apples-to-apples now that https://github.com/wikimedia/wikipedia-ios/pull/3230/commits/b296e400077f0d68f6e9f638ef8b10e0e797717f is using `self.delegate.navigationBar.visibleHeight`